smithy-diff/src/test/java/software/amazon/smithy/diff/evaluators/ChangedShapeTypeTest.java

```diff
@@ -59,4 +62,101 @@ public void ignoresExpectedSetToListMigration() {
 
         assertThat(TestHelper.findEvents(events, "ChangedShapeType"), empty());
     }
+
+    @Test
+    public void ignoresExpectedStringWithEnumTraitToEnumMigration() {
+        String conversionId = "foo.baz#Baz";
+        EnumTrait enumTrait = EnumTrait.builder()
+                .addEnum(EnumDefinition.builder()
+                        .name("FOO")
+                        .value("FOO")
+                        .build())
+                .build();
```

Comment:
Just a nit about consistency, I think is better to use all the same patterns across to board while writing code, i.e., above you inline the builders whereas here you create separate variables, above the id is just put into the builder whereas here the id is in a variable. Keeping consistency reduces the cognitive load as it makes it easier to pattern matching the constructs instead of having to re-parse every time you find a new method or jump to a different class.